src/main/resources/content/Language.properties

```diff
@@ -0,0 +1 @@
+category.sample=Sample

```

Comment:
what does this file do? Why is it called language properties when there is only a category?

Reply:
This file is used by Liferay as a dictionary, for example to define longer descriptions or categories more easily.
Here the file is not really important at first, but it makes the following difference: 

A portlet that has the Javax property `"com.liferay.portlet.display-category=category.sample"` in its `@Component`annotation, but no _Language.properties_ with the `entry category.sample=Sample` will end up in a different widget category than a portlet that has this entry. But both categories will be displayed as _SAMPLE_...

**A little visualized example:**
![categories](https://user-images.githubusercontent.com/65333616/96841373-03dfca00-144c-11eb-8f68-627b91cd68cf.png)